### ISSUE TITLE
k3s bitbake recipe

### ISSUE THREAD

**Issue by outbackdingo** [HamburgChimps/minimal-k3s#1]
i see you started the k3s recipe however i dont see an actual bitbake recipe for k3s ?
i do find the kernel and config recipes

**Comment by chopmann** [comment]
@outbackdingo Yeah, I changed employers and forgot to update chimps repo. I'm on vacation for a week and I'll get my yocto pipeline running after that. What I did not manage to do, still thinking about is the "auto-join" feature and flagging one as master and the other as a client.

**Comment by acharaj** [comment]
it's true that we don't have the recipe for k3s binary itself. also, why we need to add meta-minimal-k3s; it should just be minimal-k3s in the bblayer.conf file. @chopmann is this repo still working? do we just need to add the layer to the list of layers ?

**Comment by chopmann** [comment]
tbh - haven't used this in a while.
It set's the right kernel flags and other stuff needed to run "containers". It does not deploy the binary, nor does it compile the code.
You could get away with reusing the "docker-layers" and just wget-ing the k3s binary.
